services/hh/processor.go

```diff
@@ -144,8 +157,25 @@ func (p *Processor) Process() error {
 	p.mu.RLock()
 	defer p.mu.RUnlock()
 
-	res := make(chan error, len(p.queues))
-	for nodeID, q := range p.queues {
+	// Only process queues for active nodes.
+	var activeQueues map[uint64]*queue
+	if p.MetaProvider == nil {
```

Comment:
Can this actually happen?  There is always a meta store (remote or local) on each server right?